python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -182,7 +182,8 @@ traffic-policies
             log
 {%                             endif %}
 {%                         endif %}
-{%                     elif version is arista.avd.defined('ipv6') %}
+{%                     endif %}
+{%                     if version is arista.avd.defined('ipv6') %}
```

Comment:
Since in for loop version will be either ipv4 or ipv6 so no need of elif condition. It was causing the coverage warning, hence converted into if condition